fog/sql_recovery_db/src/lib.rs

```diff
@@ -588,10 +588,13 @@ impl SqlRecoveryDb {
             // errors.
             Err(Error::Orm(diesel::result::Error::DatabaseError(
                 diesel::result::DatabaseErrorKind::UniqueViolation,
-                _,
-            ))) => Ok(AddBlockDataStatus {
-                block_already_scanned_with_this_key: true,
-            }),
+                details,
```

Comment:
:question: There are should be no sensitive info in the error (`details`), correct?

Reply:
Yup, nothing in this DB is sensitive

Reply:
https://docs.rs/diesel/latest/diesel/result/trait.DatabaseErrorInformation.html